attest/certification_test.go

```diff
@@ -274,3 +276,72 @@ func testKeyCertification(t *testing.T, tpm *TPM) {
 		})
 	}
 }
+
+func TestKeyActivationTPM20(t *testing.T) {
+	sim, tpm := setupSimulatedTPM(t)
+	defer sim.Close()
+
+	ak, err := tpm.NewAK(nil)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
nit: please include good error messages. it really helps triage failures

t.Fatalf("creating new tpm: %v", err)

(this comment applies to the rest of this function)

Reply:
Sounds good, added more descriptive errors throughout this function